workflow/scripts/quality-filter.py

```diff
@@ -77,11 +78,32 @@ def get_n_share(contig_paths: List[str]) -> dict:
     return n_share_dict
 
 
+def get_include_rki(sample, includeflag):
+    """Extracts the information, whether the sample should be added to the rki files
+    or not out of the samples.csv file.
+
+    Args:
+        sample: List of samples for one specific date
+        includeflag: List of includeflag entries for the samples of one specific date
+
+    """
+
+    names_df = pd.DataFrame(
+        list(zip(sample, includeflag)),
+        columns=["sample_name", "include_in_high_genome_summary"],
+    )
+    include_dict = names_df.set_index("sample_name").T.to_dict("records")[0]
+
+    return include_dict
+
+
```

Comment:
Do we still need all the changes in the `quality_filter.py`? It thought all the functionally should move to the [`generate-high-quality-report.py`](https://github.com/IKIM-Essen/uncovar/blob/master/workflow/scripts/generate-high-quality-report.py).